testing/src/test/scala/org/http4s/testing/Http4sLegacyMatchers.scala

```diff
@@ -24,6 +24,7 @@ import org.http4s.headers._
 import org.http4s.util.CaseInsensitiveString
 import org.specs2.matcher.{RunTimedMatchers => Specs2RunTimedMatchers, _}
 
+@deprecated("Use Http4sSuite and munit-cats-effect-instead", "0.21.6")
```

Comment:
Wait, these are in `src/test`.  We should be able to just eliminate them, no?

Also, next one is 0.21.16, not 0.21.6.

Reply:
I'd be happy to delete them 😀